Paws4Me/Paws4Me/Controllers/Logic/SignInViewController.swift

```diff
@@ -51,8 +51,8 @@ class SignInViewController: UIViewController, Validation {
 
 func addErrorBorderBoth(_ textField1: UITextField, _ textField2: UITextField,
                         bottomLine: CALayer, bottomLine2: CALayer) {
-    bottomLine.backgroundColor = UIColor.myAppError.cgColor
-    bottomLine2.backgroundColor = UIColor.myAppError.cgColor
+    bottomLine.backgroundColor = UIColor.init(named: "primaryError")?.cgColor
```

Comment:
So this still needs to be in your style kit 😄 see previous comment.

Reply:
I have already resolved this in my last commit

Reply:
Why is this still using the colour directly then? 😄

Reply:
It is not?

<img width="602" alt="Screenshot 2022-03-08 at 07 39 12" src="https://user-images.githubusercontent.com/98941706/157173482-abb370d1-ba08-46c6-b1c7-a0b4034c6ca3.png">

Reply:
That is weird as my code on this branch shows UIColor.myAppError.cgColor

Reply:
I just added a space and committed my code and now it is reflecting the above mentioned change

Reply:
<img width="588" alt="Screenshot 2022-03-08 at 07 44 05" src="https://user-images.githubusercontent.com/98941706/157174069-3ee6d289-ec5f-4002-92cb-875fe4f7b668.png">